portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -31,13 +34,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         // Delete everything from previous step.
     }
 
+    @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
         // Get the input from the user.
         String comment = request.getParameter("comment-input");        
         
-        // Respond with the result.
-        response.setContentType("text.html;");
-        response.getWriter().println(comment);
+        Entity taskEntity = new Entity("Task");
+        taskEntity.setProperty("comment-input", comment);
+
+        // Stores the user comment in datastore.
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Does it make sense to have the datastore be a member variable of the DataServlet class?

i.e. at the top of this class

private final DatastoreService datastore;

public DataServlet() {
    super();
    datastore = DatastoreServiceFactory.getDatastoreService();
}

Reply:
It might make sense if I have to use the datastore object multiple times in the DataServlet class. I'll add it in case I need to use it again in another step.